packit/api.py

```diff
@@ -786,3 +787,54 @@ def clean(self):
         """ clean up stuff once all the work is done """
         # command handlers have nothing to clean
         logger.debug("PackitAPI.cleanup (there are no objects to clean)")
+
+    def validation(self) -> str:
```

Comment:
the common practice in python is to use verbs in method names instead of nouns, so this would be `validate`